src/backend/cdb/cdbllize.c

```diff
@@ -186,11 +186,9 @@ cdbparallelize(PlannerInfo *root,
 	switch (query->commandType)
 	{
 		case CMD_SELECT:
-			if (query->utilityStmt != NULL &&
```

Comment:
Might it be possible to create a different commandType here, instead of using the intoClause? That might allow the parallelization code to work as designed, without diverging from upstream as much.